README.md

```diff
@@ -112,6 +112,17 @@ The [graphic](#plot) at the top of the README is a visualization of this solutio
 Note that in this example, we do not model support or load-bearing constraints for cases.
 Therefore, it is possible that larger cases are packed over smaller cases without having full support.
 
+## Problem Description
+The objective included in the 3D-bin packing problem minimizes the height of the cases in the bins, 
```

Comment:
If you don't want to use the suggestion below, I'd at least rephrase this to, "The objective of the 3D-bin packing problem is to minimize ..." (that "included" is not providing much clarity)

Reply:
Thanks Joel, I added the other suggestions

Reply:
Just FYI, GitHub allows one to accumulate all the wanted suggested updates into a single commit